sqladmin/application.py

```diff
@@ -36,28 +35,39 @@ def __init__(
         base_url: str = "/admin",
         title: str = "Admin",
         logo_url: str = None,
+        template_path: str = None,
     ) -> None:
         self.app = app
         self.engine = engine
         self.base_url = base_url
-        self._model_admins: List["ModelAdmin"] = []
-
-        self.templates = Jinja2Templates("templates")
-        self.templates.env.loader = ChoiceLoader(
-            [
-                FileSystemLoader("templates"),
-                PackageLoader("sqladmin", "templates"),
-            ]
-        )
-        self.templates.env.globals["min"] = min
-        self.templates.env.globals["zip"] = zip
-        self.templates.env.globals["admin_title"] = title
-        self.templates.env.globals["admin_logo_url"] = logo_url
-        self.templates.env.globals["model_admins"] = self.model_admins
-        self.templates.env.globals["is_list"] = lambda x: isinstance(x, list)
+        self.title = title
+        self.logo_url = logo_url
+        self.template_path = template_path
+        self._model_admins: List["ModelView"] = []
+
+        self.templates = self.init_templating_engine()
+
+    def init_templating_engine(self) -> Jinja2Templates:
+        templates = Jinja2Templates("templates")
+        loaders = [
+            FileSystemLoader("templates"),
```

Comment:
```suggestion
            FileSystemLoader(self.templates_dir),
```

Should be ok. Since we can default it to `templates`.